crypto/ec/ecdsa_ossl.c

```diff
@@ -298,7 +298,7 @@ ECDSA_SIG *ossl_ecdsa_sign_sig(const unsigned char *dgst, int dgst_len,
         }
         if (BN_is_zero(s)) {
             /*
-             * if kinv and r have been supplied by the caller don't to
+             * if kinv and r have been supplied by the caller don't do
              * generate new kinv and r values
```

Comment:
remove the do?

Reply:
maybe, I am not a native English speaker...
should the sentence start with capital letter, and end with a period?

Reply:
ok, so I just remove the "do".